parameter_files/ProjectParameters_PA2024CH.yml

```diff
@@ -0,0 +1,66 @@
+default:
+
+    paths:
+        data_location_ext: ../pacta-data/2023Q4/
+
+    reporting:
+        project_report_name: PA2024CH
+        display_currency: CHF
+        currency_exchange_value: 1.190405
+
+    parameters:
+        start_year: 2023
+        horizon_year: 5
+        select_scenario: WEO2023_NZE_2050
+        scenario_other: GECO2023_1.5C-Unif
+        portfolio_allocation_method: portfolio_weight
+        scenario_geography: Global
+
+    methodology:
+        has_map: TRUE
+        has_revenue: FALSE
+        has_credit: FALSE
+        inc_emissionfactors: TRUE
+
+    sectors:
+        tech_roadmap_sectors:
+            - Power
+            - Automotive
+            - Oil&Gas
+            - Coal
+        pacta_sectors_not_analysed:
+            - Steel
+            - Aviation
+            - Cement
+        green_techs:
+            - RenewablesCap
+            - HydroCap
+            - NuclearCap
+            - Hybrid
+            - Electric
+            - FuelCell
+            - Electric Arc Furnace
+        alignment_techs:
+            - RenewablesCap
+            - CoalCap
+            - Coal
+            - Oil
+            - Gas
+            - Electric
+            - ICE
+
+    scenario_sources_list:
+        - GECO2023
+        - ISF2023
+        - WEO2023
+
+    scenario_geography_list:
+        - Global
+        - GlobalAggregate
+        - NonOECD
+        - OECD
+
+    equity_market_list:
+        - GlobalMarket
+        - DevelopedMarket
+        - EmergingMarket
```

Comment:
recent data.prep includes `DevelopedMarket`, `EmergingMarket`, `GlobalMarket`, and `USMarket`. Don't think we've ever used `USMarket`, so this makes sense to me.
``` r
root_dir <- "~/Downloads"
timestamp_dir <- "2023Q4_20240228T144514Z"

equity_abcd_scenario <- readRDS(file.path(root_dir, timestamp_dir, "equity_abcd_scenario.rds"))
bonds_abcd_scenario <- readRDS(file.path(root_dir, timestamp_dir, "bonds_abcd_scenario.rds"))

sort(unique(equity_abcd_scenario$equity_market))
#> [1] "DevelopedMarket" "EmergingMarket"  "GlobalMarket"    "USMarket"
sort(unique(bonds_abcd_scenario$equity_market))
#> [1] "GlobalMarket"
```